x-pack/plugins/maps/public/classes/fields/count_agg_field.ts

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { IndexPattern } from 'src/plugins/data/public';
+import { IESAggSource } from '../sources/es_agg_source';
+import { IVectorSource } from '../sources/vector_source';
+import { AGG_TYPE, FIELD_ORIGIN } from '../../../common/constants';
+import { ITooltipProperty, TooltipProperty } from '../tooltips/tooltip_property';
+import { ESAggTooltipProperty } from '../tooltips/es_agg_tooltip_property';
+import { IESAggField, IESAggFieldParams } from './agg_field_types';
+
+// Agg without field. Essentially a count-aggregation.
+export class CountAggField implements IESAggField {
+  private readonly _source: IESAggSource;
+  private readonly _origin: FIELD_ORIGIN;
+  private readonly _label?: string;
+  private readonly _canReadFromGeoJson: boolean;
+
+  constructor({ label, source, origin, canReadFromGeoJson = true }: IESAggFieldParams) {
+    this._source = source;
+    this._origin = origin;
+    this._label = label;
+    this._canReadFromGeoJson = canReadFromGeoJson;
+  }
+
+  _getESDocFieldName(): string {
```

Comment:
This method is only used in `getRootName`. How about just removing the method and making `getRootName` return empty string?